### PR TITLE
feat: DWEM-1 Add new landing page

### PR DESCRIPTION
**Pull request by volos-codice**
- added new landing page
- improved README file
- fixed mobile view
- fixed cards date

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,10 +36,20 @@ npm run dist
 
 to compile, and copy all site files to the `dist/` folder
 
+## How I should install correct node version, npm and bower?
```
Comment (alex-codice) [inline README.md:39]: what do you think about changing from `How I should` to `How should I`?

File: src/styles/main/_landing.scss [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+  z-index: $zIndex10;
+
+  @include media($medium-screen-up) {
+    height: calc(100vh + 2em);
+  }
+
+  .landing-main-container {
+    color: $dark-gray;
+    margin: 0;
+    padding: 1em 2em 6cm 2em;
```
Comment (alex-codice) [inline src/styles/main/_landing.scss:18]: was `6cm` written intensionally?

File: README.md (hunk 1 above)
Comment (volos-codice) [reply README.md]: Changed

File: src/styles/main/_landing.scss (hunk 2 above)
Comment (volos-codice) [reply src/styles/main/_landing.scss]: Fixed

**Pull request merged**